### PR TITLE
Bug 1197444: Rename kuma/static to kuma/assets

### PR DESCRIPTION
**Pull request by openjck**
It's easy to confuse the kuma/static directory and the root static
directory, especially when the term kuma is already in the path.

Can you easily tell the difference between these?

    /path/to/dev/folder/kuma/kuma/static
    /path/to/dev/folder/kuma/static

Bleh. This change renames kuma/static to kuma/assets to make the
distinction more clear.

--

To test:

* Browse around and ensure the site generally looks good
* Repeat after [enabling production assets](https://kuma.readthedocs.org/en/latest/development.html#production-assets)

### CODE REVIEW

File: kuma/assets/css/font-awesome.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
...
...{0%{-webkit-transform:rotate(0)}100%{-webkit-transform:rotate(359deg)}}@-webkit-keyframes spin{0%{-webkit-transform:rotate(0)}100%{-webkit-transform:rotate(359deg)}}@-o-keyframes spin{0%{-webkit-transform:rotate(0)}100%{-webkit-transform:rotate(359deg)}}@keyframes spin{0%{-webkit-transform:rotate(0)}100%{-webkit-transform:rotate(359deg)}}@-moz-keyframes spin{0%{-o-transform:rotate(0)}100%{-o-transform:rotate(359deg)}}@-webkit-keyframes spin{0%{-o-transform:rotate(0)}100%{-o-transform:rotate(359deg)}}@-o-keyframes spin{0%{-o-transform:rotate(0)}100%{-o-transform:rotate(359deg)}}@keyframes spin{0%{-o-transform:rotate(0)}100%{-o-transform:rotate(359deg)}}@-moz-keyframes spin{0%{-webkit-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(359deg);transform:rotate(359deg)}}@-webkit-keyframes spin{0%{-webkit-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(359deg);transform:rotate(359deg)}}@-o-keyframes spin{0%{-webkit-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(359deg);transform:rotate(359deg)}}@keyframes spin{0%{-webkit-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(359deg);transform:rotate(359deg)}}
```
Comment (mdn-services) [inline kuma/assets/css/font-awesome.css]: Font unicode-range subsetting not supported by: IE (10,11), Firefox (38,39), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)

File: kuma/assets/css/wiki-wysiwyg.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+pre[class^='brush:bash']:before, pre[class^='brush: bash']:before { content: 'Bash' }
+pre[class^='brush:cpp']:before, pre[class^='brush: cpp']:before { content: 'C++' }
+pre[class^='brush:java']:before, pre[class^='brush: java']:before { content: 'Java' }
+pre[class^='brush:html']:before, pre[class^='brush: html']:before { content: 'HTML' }
+pre[class^='brush:php']:before, pre[class^='brush: php']:before { content: 'PHP' }
+pre[class^='brush:python']:before, pre[class^='brush: python']:before { content: 'Python' }
+pre[class^='brush:sql']:before, pre[class^='brush: sql']:before { content: 'SQL' }
+pre[class^='brush:xml']:before, pre[class^='brush: xml']:before { content: 'XML' }
+li[data-default-state='open'] > a:after{content:'(defaults to "open")';color:#666;font-size:12px;font-size:.857rem;font-family:arial,sans-serif;font-style:italic;display:inline-block;padding-left:6px}details{position:relative;display:block;border:2px dotted #000;padding:3px;}details:before{font-size:10px;font-size:.714rem;content:'details';position:absolute;top:-17px;left:-2px;z-index:10;display:block;padding:0 2px;background:#000;color:#fff}
```
Comment (mdn-services) [inline kuma/assets/css/wiki-wysiwyg.css]: rem (root em) units not supported by: IE (10)

Comment (mdn-services) [inline kuma/assets/css/wiki-wysiwyg.css]: rem (root em) units not supported by: IE (10)

**Comment by openjck** [comment]
Heh. Discord added some comments.

This change just moves the files, so I'm not worried about them.

**Comment by groovecoder** [comment]
Checking this out now. And yes, the static thing is totally confusing to me too.

**Comment by groovecoder** [comment]
On a simple checkout + runserver, I get 404s for all the CSS at first. Had to run `./scripts/compile-stylesheets` first. That will teach me to use foreman I guess.

**Comment by groovecoder** [comment]
Unrelated to this PR(?) - mozorg CDN is missing a bunch of .woff2 files: http://screencast.com/t/kGW9xHwCL

**Comment by groovecoder** [comment]
Intern tests run fine in both local static mode and production static mode. (Nice way to verify was to see the requests for the static assets show up in runserver when running with `DEBUG=True` and then see them disappear when running with `DEBUG=False`).

nits?

* Our [production assets docs instruction to restart apache](https://kuma.readthedocs.org/en/latest/development.html#production-assets) could be explicit: `sudo apache2ctl restart` ?

* I noticed I have/had a `static/assets/` directory in my box, but it seems unnecessary? I deleted it completely and all tests still run fine in both modes. Is there some `static/` cleanup to do after this change?

**Comment by openjck** [comment]
> Unrelated to this PR(?) - mozorg CDN is missing a bunch of .woff2 files: http://screencast.com/t/kGW9xHwCL

Different PR, yeah. Let's take the discussion to IRC. cc @darkwing 

> Our production assets docs instruction to restart apache could be explicit: sudo apache2ctl restart ?

Updated

> I noticed I have/had a static/assets/ directory in my box, but it seems unnecessary? I deleted it completely and all tests still run fine in both modes. Is there some static/ cleanup to do after this change?

Hm. I can't seem to reproduce that. Can you think of another way it might have ended up there? I don't want to write it off, but it also shouldn't cause any problems if it is there, so maybe we can just keep an eye on it.

**Pull request merged**